src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -133,8 +135,11 @@ export function getConverterData(layoutInfo=getLayouInfo()) {
     const converterId = get(layoutInfo, [LC.MISSION_DATA, LC.META_MISSION]);
     return converterId && getConverter(converterId);
 }
+export function getConvertId(layoutInfo=getLayouInfo()) {
+    return get(layoutInfo, [LC.MISSION_DATA, LC.META_MISSION]);
```

Comment:
this logic is used in many places and is referred to as converterId.  if this is a convenience function, you should name is as getConverterId, not getConvertId.  changing all other places to use this function may be a good idea if it's truly needed.